react-native-purchases-ui/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "rootDir": ".",
+    "paths": {
+      "react-native-purchases-ui": ["./src/index"]
+    },
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "lib": ["esnext"],
+    "module": "esnext",
```

Comment:
I am not sure about this, it's different from the tsconfig in the root, but this was the default in the template

Reply:
If we generate both js code for commonjs and esnext with bob, I believe it should be compatible yeah... But definitely it would be good to get some beta testers trying this.